paymentsheet/src/main/java/com/stripe/android/link/ui/wallet/WalletViewModel.kt

```diff
@@ -262,6 +260,7 @@ internal class WalletViewModel @Inject constructor(
             selectedPaymentDetails = selectedPaymentDetails,
             linkAccount = linkAccount,
             cvc = cvc,
+            billingPhoneNumber = linkAccount.unredactedPhoneNumber,
```

Comment:
On the wallet, our only phone source without re-collecting is the link account. If there's no link account phone, the confirm details screen would pop up before reaching this.